test/functional/home_controller_test.rb

```diff
@@ -50,6 +45,7 @@ def setup
     UserSession.create(users(:bob))
     get :dashboard
     assert_response :success
+    assert_select "a.btn.btn-xs.btn-default.btn-flag-spam-#{nodes(:one).id}", :href => "mailto:moderators@publiclab.org?subject=Reporting+spam+on+Public+Lab&body=Hi,+I+found+this+item+that+looks+like+spam+or+needs+to+be+moderated:+Canon+A1200+IR+conversion+at+PLOTS+Barnraising+at+LUMCON+https://publiclab.org/n/1+by+https://publiclab.org/profile/jeff+thanks!"
```

Comment:
This syntax is depreciated ! 
Found here : 
https://github.com/rails/rails/issues/19098
http://edgeguides.rubyonrails.org/4_2_release_notes.html#assert-select

Reply:
Also T is capital in thanks .